saleor/graphql/attribute/mutations/attribute_bulk_create.py

```diff
@@ -0,0 +1,525 @@
+                        path="values",
+                        message=(
+                            "Values cannot be used with " f"input type {input_type}.",
+                        ),
+                        code=AttributeBulkCreateErrorCode.INVALID.value,
+                    )
+                )
+                continue
+
+            cls._validate_value(
+                value_data,
+                input_type,
+                slugs_list,
+                value_index,
+                attribute_index,
+                index_error_map,
+            )
+
+            cleand_values.append(value_data)
+
+        return cleand_values
+
+    @classmethod
+    def _validate_value(
+        cls,
+        value_data,
+        input_type,
+        slugs_list,
+        value_index,
+        attribute_index,
+        index_error_map,
+    ):
+        name = value_data.get("name")
+        is_swatch_attr = input_type == AttributeInputType.SWATCH
+
+        if is_swatch_attr and value_data.get("value") and value_data.get("file_url"):
+            index_error_map[attribute_index].append(
+                AttributeBulkCreateError(
+                    path=f"values.{value_index}",
+                    message=(
+                        "Cannot specify both value and file for swatch attribute."
+                    ),
+                    code=AttributeBulkCreateErrorCode.INVALID.value,
+                )
+            )
+        else:
+            if any([value_data.get(field) for field in ONLY_SWATCH_FIELDS]):
+                index_error_map[attribute_index].append(
+                    AttributeBulkCreateError(
+                        path=f"values.{value_index}",
+                        message=(
+                            "Cannot define value, file and contentType fields "
+                            "for not swatch attribute.",
+                        ),
+                        code=AttributeBulkCreateErrorCode.INVALID.value,
+                    )
+                )
+
+        slug_value = prepare_unique_slug(slugify(unidecode(name)), slugs_list)
+        value_data["slug"] = slug_value
+        slugs_list.append(slug_value)
+
+        return value_data
+
+    @classmethod
+    def _generate_slug(cls, cleaned_input, existing_slugs):
+        name = cleaned_input.get("name")
+        slug = slugify(unidecode(name))
+
+        unique_slug = prepare_unique_slug(slug, existing_slugs)
+        existing_slugs.add(unique_slug)
+
+        return unique_slug
+
+    @classmethod
+    def create_attributes(cls, info, cleaned_inputs_map, index_error_map):
+        instances_data_and_errors_list = []
+
+        for index, cleaned_input in cleaned_inputs_map.items():
+            if not cleaned_input:
+                instances_data_and_errors_list.append(
+                    {"instance": None, "errors": index_error_map[index]}
+                )
+                continue
+            try:
+                values = []
```

Comment:
Maybe we could limit the body of try/except here? I mean, can we leave here a part that already can raise the exception? For example: I don't think the line like `values_data = cleaned_input.pop("values",None)` or `instance = models.Attribute()` can raise an exception. correct?